rest/proxy/proxy.go

```diff
@@ -102,13 +102,8 @@ func newDirector(ctx context.Context, originalRequestData *goa.RequestData, targ
 		if requestID != "" {
 			req.Header.Set("X-Request-ID", requestID)
 		}
-		// Remove all *Forwarded* headers from the request because they belong to the original request
-		// and we should not pass them along
-		for key := range req.Header {
-			if strings.Contains(key, "Forwarded") {
-				req.Header.Del(key)
-			}
-		}
+
+		req.Header.Set("X-Forwarded-proto", scheme)
```

Comment:
req.Header.Set("X-Forwarded-proto", scheme) should not be necessary. It's already in the request if we remove lines 105-111

Reply:
That's what I thought too - but noticed it wasn't being set locally at least. So thought we could set it explicitly anyway. Or, move it to the `else` block : https://github.com/sbose78/almighty-core/blob/9cb53e327ae58c3b51e6a657d1bf82185914caf9/rest/proxy/proxy.go#L77

Reply:
This header is not set by our code. It's set by OpenShift. Internally http is used in the cluster. But the original requests are using https. OS uses this header to keep the schema used in the original request. So, we just don't delete this header it should be enough.

Reply:
OK, not setting it anymore explicitly -
https://github.com/fabric8-services/fabric8-wit/pull/1700/commits/e9269cf826e49a8b153ec5dbe56fbd11d761e2ea